packages/react-components/package.json

```diff
@@ -13,12 +13,15 @@
   "license": "MIT",
   "scripts": {
     "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
     "bundle:storybook": "just-scripts storybook:build",
+    "chromatic": "npx chromatic@5.6.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes",
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook"
+    "start": "just-scripts dev:storybook",
+    "build-storybook": "just-scripts storybook:build"
```

Comment:
This is identical to the `bundle:storybook` script above (it might have been added after the PR was created). Probably best to stick with one or the other?